fe1-web/store/reducers/__tests__/SocialReducer.test.ts

```diff
@@ -16,12 +16,13 @@ import {
 } from '../SocialReducer';
 
 const mockPublicKey = new PublicKey(keyPair.publicKey);
-const org = mockPublicKey;
-const name = 'MyLao';
-const mockLaoIdHash: Hash = Hash.fromStringArray(
-  org.toString(), new Timestamp(160000000).toString(), name,
+export const org = mockPublicKey;
+export const mockLaoName = 'MyLao';
+export const mockLaoCreationTime = new Timestamp(160000000);
+export const mockLaoIdHash: Hash = Hash.fromStringArray(
+  org.toString(), mockLaoCreationTime.toString(), mockLaoName,
 );
-const mockLaoId: string = mockLaoIdHash.toString();
+export const mockLaoId: string = mockLaoIdHash.toString();
```

Comment:
Same comment here, why should all these exported constants be in SocialReducer ?